Rye/Rye/RyeViewController.swift

```diff
@@ -64,14 +65,16 @@ public class RyeViewController: UIViewController {
         self.dismissMode = dismissMode
         self.viewType = viewType
         self.position = position
-        
+
         switch viewType {
         case .standard(let configuration):
             animationDuration = configuration?[Rye.Configuration.Key.animationDuration] as? TimeInterval ?? 0.3
             animationType = configuration?[Rye.Configuration.Key.animationType] as? Rye.AnimationType ?? .slideInOut
-        case .custom(_, let animationType):
-            animationDuration = 0.3
-            self.animationType = animationType
+            ignoreSafeAreas = configuration?[Rye.Configuration.Key.ignoreSafeAreas] as? Bool ?? false
+        case .custom(_, let configuration):
+            animationDuration = configuration?[Rye.Configuration.Key.animationDuration] as? TimeInterval ?? 0.3
+            animationType = configuration?[Rye.Configuration.Key.animationType] as? Rye.AnimationType ?? .slideInOut
+            ignoreSafeAreas = configuration?[Rye.Configuration.Key.ignoreSafeAreas] as? Bool ?? false
         }
```

Comment:
So now we have this little beauty where we do the same thing in both cases.

Which leads to potential future improvements 😄 

We should probably separate `viewType` and `configuration` so that `viewType` is just a "dumb" enum looking like so:

```
    public enum ViewType {
        case standard
        case custom(UIView)
    }
```

And the `RyeConfiguration` is then passed as another parameter when you create a view.

This is a somewhat larger - and backward compatibility breaking - operation though so maybe we shouldn't release that just now?